[2025-02-08T03:55:40.104+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T03:55:40.123+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T03:55:36.966644+00:00 [queued]>
[2025-02-08T03:55:40.135+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T03:55:36.966644+00:00 [queued]>
[2025-02-08T03:55:40.136+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-08T03:55:40.152+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extraer_logs> on 2025-02-08 03:55:36.966644+00:00
[2025-02-08T03:55:40.162+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1711) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-08T03:55:40.164+0000] {standard_task_runner.py:72} INFO - Started process 1713 to run task
[2025-02-08T03:55:40.164+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Incident_Report_Generation', 'extraer_logs', 'manual__2025-02-08T03:55:36.966644+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/Incident_Report_Generation.py', '--cfg-path', '/tmp/tmpht4ctfgt']
[2025-02-08T03:55:40.165+0000] {standard_task_runner.py:105} INFO - Job 52: Subtask extraer_logs
[2025-02-08T03:55:40.224+0000] {task_command.py:467} INFO - Running <TaskInstance: Incident_Report_Generation.extraer_logs manual__2025-02-08T03:55:36.966644+00:00 [running]> on host 0b0ecc2e9e80
[2025-02-08T03:55:40.319+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Incident_Report_Generation' AIRFLOW_CTX_TASK_ID='extraer_logs' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T03:55:36.966644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-08T03:55:36.966644+00:00'
[2025-02-08T03:55:40.321+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-08T03:55:40.322+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-08T03:55:40.322+0000] {logging_mixin.py:190} INFO - Current task name:extraer_logs state:running start_date:2025-02-08 03:55:40.124698+00:00
[2025-02-08T03:55:40.323+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation and current dag run status:running
[2025-02-08T03:55:40.324+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T03:55:40.335+0000] {base.py:84} INFO - Retrieving connection 'mysql_default'
[2025-02-08T03:55:41.366+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T03:55:41.376+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T03:55:41.377+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Incident_Report_Generation, task_id=extraer_logs, run_id=manual__2025-02-08T03:55:36.966644+00:00, execution_date=20250208T035536, start_date=20250208T035540, end_date=20250208T035541
[2025-02-08T03:55:41.391+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-08T03:55:41.391+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-08T03:55:41.392+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation queued_at:2025-02-08 03:55:37.022448+00:00
[2025-02-08T03:55:41.393+0000] {logging_mixin.py:190} INFO - Task hostname:0b0ecc2e9e80 operator:PythonOperator
[2025-02-08T03:55:41.430+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T03:55:41.470+0000] {taskinstance.py:3895} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-02-08T03:55:41.474+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
